doc/cla/individual/hadialawieh.md

```diff
@@ -0,0 +1,10 @@
+Lebanon, 2024-03-06
```

Comment:
As employees, we shouldn't add your name in the CLA, it's for external contributors.

You should instead link your odoo address to your account in order to remove the runbot warning:
<https://github.com/odoo/enterprise/wiki/Git#global-settings>
Once you do that, you can remove this file and the warning shouldn't pop up :)